.github/workflows/integration-tests.yml

```
@@ -45,14 +45,18 @@ jobs:
 
       # load docker images from build jobs
       - name: Load images from artifacts
-        uses: actions/download-artifact@v3
+        uses: actions/download-artifact@v4
+        with:
+          path: built-docker-image
+          pattern: built-docker-image-*
+          merge-multiple: true
 
       - name: Load docker images
         timeout-minutes: 10
         run: |-
-             docker load -i nginx/nginx-${{ matrix.os }}_img
-             docker load -i django/django-${{ matrix.os }}_img
-             docker load -i integration-tests/integration-tests-debian_img
+             docker load -i built-docker-image/nginx-${{ matrix.os }}_img
+             docker load -i built-docker-image/django-${{ matrix.os }}_img
+             docker load -i built-docker-image/integration-tests-debian_img
```

Will this actually trim the `built-docker-image-` prefix from these image names, or do we need something like this instead? Wasn't clear to me from the migration guide...

```suggestion
             docker load -i built-docker-image/built-docker-image-nginx-${{ matrix.os }}_img
             docker load -i built-docker-image/built-docker-image-django-${{ matrix.os }}_img
             docker load -i built-docker-image/built-docker-image-integration-tests-debian_img
```

---

I don't believe the prefix is trimmed. Do we need the prefix to be trimmed?

---

Also I didn't realize when I left the last comment, but the names of the images themselves do not have a `built-docker-image-` prefix. The name of the image stored is captured in the `path` entry:
```
path: ${{ matrix.docker-image }}-${{ matrix.os }}_img
```

This value is unchanged in this PR

---

Ahh, I think I mixed up the [`with` (artifact name)](https://github.com/Maffooch/django-DefectDojo/blob/ce0763271668b042b51489a6b730d700bbcd33ee/.github/workflows/build-docker-images-for-testing.yml#L54) and the [`path`](https://github.com/Maffooch/django-DefectDojo/blob/ce0763271668b042b51489a6b730d700bbcd33ee/.github/workflows/build-docker-images-for-testing.yml#L55)